controllers/controller/devworkspacerouting/devworkspacerouting_controller.go

```diff
@@ -115,7 +115,6 @@ func (r *DevWorkspaceRoutingReconciler) Reconcile(req ctrl.Request) (ctrl.Result
 		DevWorkspaceId: instance.Spec.DevWorkspaceId,
 		Namespace:      instance.Namespace,
 		PodSelector:    instance.Spec.PodSelector,
-		RoutingSuffix:  instance.Spec.RoutingSuffix,
```

Comment:
You removed assigning DevWorkspaceMetadata.RoutingSuffix but it seems to be still used. I wonder if it should be fully removed from struct and evaluated by routing impl in place where it's needed
![Screenshot_20210419_100146](https://user-images.githubusercontent.com/5887312/115194534-441bbe80-a0f6-11eb-9be0-1b76fa2b5213.png)